lib/assessment.js

```diff
@@ -8,32 +8,73 @@
 // Question 1
 // Write a callback for reduce that selects
 // the minimum value in the array
-var min = function min() {
+var min = function min(previousValue, currentValue, index, array) {
+    var result = previousValue;
+    if(currentValue < result) result = currentValue;
+    return result;
+
 
 };
 
+var array = [5, 3, 2, 4]
+console.log(array.reduce(min))
+
 // Question 2
 // Write a callback for reduce that produces
 // the sum of the values in the array
-var sum = function sum() {
+var sum = function sum(previousValue, currentValue, index, array) {
+       var result = previousValue;
+     result += currentValue;
+    return result;
+
```

Comment:
Good. The code is correct. All I can ask you to do here is use consistent indentation for future reading.